### PR TITLE
improved documentation, fixed bug with postgres in docker-compose

### PR DESCRIPTION
**Pull request by geekscruff**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -42,21 +42,25 @@ services:
...
     env_file:
       - .env
     healthcheck:
-      test: "CMD echo 'SELECT 1' | POSTGRES_PASSWORD=${POSTGRES_PASSWORD} psql --host 127.0.0.1 --username #{POSTGRES_USER} --dbname #{POSTGRES_DB} --quiet --no-align --tuples-only"
+      test: "CMD echo 'SELECT 1' | POSTGRES_PASSWORD=${POSTGRES_PASSWORD} psql --host 127.0.0.1 --username ${POSTGRES_USER} --dbname ${POSTGRES_DB} --quiet --no-align --tuples-only"
       interval: 30s
       timeout: 5s
       retries: 3
+    volumes:
+      - db:/var/lib/postgresql/data
```
Comment (anusharanganathan) [inline docker-compose.yml]: Volumes defined twice for db

File: docker-compose.yml [hunk 2]
```diff
@@ -42,21 +42,25 @@ services:
...
     networks:
       internal:
+    expose:
+      - ${POSTGRES_PORT}
```
Comment (anusharanganathan) [inline docker-compose.yml]: Is it a good idea to expose a database port? Would it need to be accessed by an application outside?

Comment (geekscruff) [reply docker-compose.yml]: Actually, thinking about it I don't think it does need to be exposed. Have removed this.

File: docker-compose.yml (hunk 1 above)
Comment (geekscruff) [reply docker-compose.yml]: Fixed!

**Pull request merged**